public/app/features/dashboard/components/GenAI/GenAIButton.tsx

```diff
@@ -1,17 +1,19 @@
 import { css } from '@emotion/css';
-import React, { useEffect, useState } from 'react';
+import React from 'react';
 
 import { GrafanaTheme2 } from '@grafana/data';
+import { llms } from '@grafana/experimental';
 import { Button, Spinner, useStyles2, Link, Tooltip } from '@grafana/ui';
 
-import { Message, generateTextWithLLM, isLLMPluginEnabled } from './utils';
+import { useOpenaiStreamHook } from './hooks';
```

Comment:
```suggestion
import { useOpenaiStream } from './hooks';
```
Do we need to include Hook in the name of the function? IMO with the prefix `useX` it should be enough.